src/quo2/screens/messages/system_message.cljs

```diff
@@ -0,0 +1,77 @@
+(ns quo2.screens.messages.system-message
+  (:require [reagent.core :as reagent]
+            [quo.react-native :as rn]
+            [quo.previews.preview :as preview]
+            [quo2.components.messages.system-message :as system-message]
+            [quo2.screens.messages.resources :as resources]))
+
+(def descriptor [{:label   "Message Type"
+                  :key     :type
+                  :type    :select
+                  :options [{:value "Message pinned"
+                             :key   :pinned}
+                            {:value "User added"
+                             :key   :added}
+                            {:value "Message deleted"
+                             :key   :deleted}]}
+                 {:label   "Message State"
+                  :key     :state
+                  :type    :select
+                  :options [{:value "Default"
+                             :key   :default}
+                            {:value "Pressed"
+                             :key   :pressed}
+                            {:value "Landed"
+                             :key   :landed}]}
```

Comment:
these are probably actions on the component, and shouldn't be controlled by preview screen controllers

Reply:
Done